src/include/daos/tests_lib.h

```diff
@@ -241,6 +241,58 @@ int dmg_pool_create(const char *dmg_config_file,
 int dmg_pool_destroy(const char *dmg_config_file,
 		     const uuid_t uuid, const char *grp, int force);
 
+/**
+ * Exclude an entire rank or a target on that rank from a pool.
+ *
+ * \param dmg_config_file
+ *			[IN]	DMG config file
```

Comment:
FYI, this annotation style is not doxygen, please don't use it.  We still need to convert existing files like this one to the proper format.